src/bin/cargo-flamegraph.rs

```diff
@@ -67,6 +67,10 @@ struct Opt {
     #[structopt(long = "root")]
     root: bool,
 
+    /// Sampling frequency
+    #[structopt(short = "F", long = "freq")]
+    frequency: Option<String>,
```

Comment:
This should not accept any string here, probably something like `u16` is sufficient